intake/catalog/remote.py

```diff
@@ -109,16 +111,20 @@ def open_remote(url, entry, container, user_parameters, description, http_args,
         if 'plugin' in response:
             # Direct access
             source = plugin_registry[response['plugin']](**response['args'])
+            source.description = description
         else:
             # Proxied access
+            response.pop('container')
+            response.update({'name': entry, 'parameters': user_parameters})
             if container == 'catalog':
-                # Propagate the page_size setting into nested Catalogs.
-                response['page_size'] = page_size
-                response.pop('container')
-            source = container_map[container](
-                url, http_args, parameters=user_parameters,
-                name=entry, **response)
-        source.description = response.get('description', description)
```

Comment:
I replaced this with `source.description = description` above because the response only _has_ a description if it is a direct-access response, so there was no need to do it in the proxied-access case as well.